src/components/Canvas.tsx

```diff
@@ -42,18 +44,24 @@ const Canvas = () => {
     });
   };
 
+  const handleKeyPress = (key: string) => {
+    if (key === 'Delete') deleteElements();
+  };
+
   return (
     <svg
       ref={svgContainerRef}
       onMouseDown={(e) => handleMouseDown(e)}
       onMouseMove={(e) => handleMouseMove(e)}
       onMouseUp={onMouseUp}
+      onKeyDown={(e) => handleKeyPress(e.key)}
       className="border-4 border-green-600"
       preserveAspectRatio="xMinYMin meet" //for the SVG container to be on the entire screen, while the elements inside kept the proportions and x=0, y=0 viewBox started from the upper left corner
       viewBox="0 0 1920 1080"
       width="100%"
       height="100%"
       xmlns="http://www.w3.org/2000/svg"
+      tabIndex={0}
```

Comment:
why do you need tabIndex here?

Reply:
to focus on the canvas.
I need it to listen key press when deleting element